pkg/download/download.go

```diff
@@ -96,13 +96,19 @@ func getAPIList(downloadSpecificAPI string) (filterAPIList map[string]api.Api, e
 }
 
 //creates the project and downloads the configs
-func downloadConfigFromEnvironment(environment environment.Environment, basepath string, listApis map[string]api.Api) (err error) {
+func downloadConfigFromEnvironment(fileManager afero.IOFS, environment environment.Environment, basepath string, listApis map[string]api.Api) (err error) {
 	projectName := environment.GetId()
 	path := filepath.Join(basepath, projectName)
-	creator := files.NewDiskFileCreator()
 
 	util.Log.Info("Creating base project name %s", projectName)
-	fullpath, err := creator.CreateFolder(path)
+	exist, err := afero.Exists(fileManager.Fs, path)
+	if !exist && err == nil {
+		err = fileManager.Mkdir(path, 0777)
+	}
+	if err != nil {
```

Comment:
Please move this if before the one on line 105.
You can then drop the second condition for the if on 105.